recipes/libxml2/all/conanfile.py

```diff
@@ -23,7 +23,7 @@ class Libxml2Conan(ConanFile):
     topics = "xml", "parser", "validation"
     homepage = "https://gitlab.gnome.org/GNOME/libxml2/-/wikis/"
     license = "MIT"
-
+    package_type = "library"
```

Comment:
there is already a package_type above

Reply:
@SpaceIm 
Thank you.
Oh, it's my mistake. remove it.